### PR TITLE
Changes for MultiNode

### PR DESCRIPTION
**Pull request by BenyH**

### CODE REVIEW

**Comment by ozlevka-ericom** [comment]
Beny!
Stop edit bash scripts on Windows. It's brake the code...

File: Setup/docker-compose_dev.yml [hunk 1]
```diff
@@ -4,29 +4,56 @@ version: "3.3"
...
+      - /var/run/docker.sock:/var/run/docker.sock
+    networks:
+      - shield
+    hostname: netdata
+    ports:
+      - 8282:19999
+    cap_add:
+      - SYS_PTRACE
+    volumes:
+      - /proc:/host/proc:ro
```
Comment (ozlevka-ericom) [inline Setup/docker-compose_dev.yml]: You have mount proc directory twice it's broke installation.
I fix it. See my commit later

File: Setup/ericomshield-repo.sh [hunk 2]
```diff
@@ -1,37 +1,38 @@
...
-ES_repo_dev_yml="https://raw.githubusercontent.com/EricomSoftwareLtd/Shield/$BRANCH/Setup/docker-compose_dev.yml"
-ES_repo_pocket_yml="https://raw.githubusercontent.com/EricomSoftwareLtd/Shield/$BRANCH/Setup/docker-compose_pocket.yml"
-ES_repo_swarm_sh="https://raw.githubusercontent.com/EricomSoftwareLtd/Shield/$BRANCH/Setup/deploy-shield.sh"
-ES_repo_swarm_dev_sh="https://raw.githubusercontent.com/EricomSoftwareLtd/Shield/$BRANCH/Setup/deployshield-dev.sh"
+#!/bin/bash
+############################################
+#####   Ericom Shield Installer        #####
+#######################################BH###
+
+BRANCH="master"
```
Comment (ozlevka-ericom) [inline Setup/ericomshield-repo.sh]: Not need declare variable twice

Comment (BenyH) [reply Setup/ericomshield-repo.sh]: But this file is also included from status.sh
maybe we need to add a condition?

Comment (ozlevka-ericom) [reply Setup/ericomshield-repo.sh]: Then remove from ericomshield-setup. Now you need remember two places where need change variable...

**Comment by ozlevka-ericom** [comment]
Hi Beny,
We need to avoid any interaction with user...
This mean we should ban EULA and IP choosing... This node is not deploy any thing and only join to cluster...

**Comment by ozlevka-ericom** [comment]
Hi Beny,
Question:
Do we need download all images with installation?

File: Setup/ericomshield-repo.sh (hunk 2 above)
Comment (ozlevka-ericom) [reply Setup/ericomshield-repo.sh]: Also need to allow receive branch as parameter. I will change, please review

**Pull request merged**